pkg/function/create_rds_snapshot.go

```diff
@@ -130,12 +132,23 @@ func createRDSSnapshot(ctx context.Context, instanceID string, dbEngine RDSDBEng
 	if err != nil {
 		return nil, errors.Wrapf(err, "Failed to create securityGroupID artifact. InstanceID=%s", instanceID)
 	}
+	// Find dbSubnetGroup
+	var dbSubnetGroup *string
+	if !isAuroraCluster(string(dbEngine)) {
+		dbSubnetGroup, e = GetRDSDbSubnetGroup(ctx, rdsCli, instanceID)
+	} else {
+		dbSubnetGroup, e = GetRDSAuroraDbSubnetGroup(ctx, rdsCli, instanceID)
+	}
+	if e != nil {
+		return nil, errors.Wrapf(e, "Failed to get dbSubnetGroup ids. InstanceID=%s", instanceID)
+	}
```

Comment:
[nit] Lets use a switch case for better readability

Reply:
Not sure if we should do that, because its would just a true false statement.

Reply:
It improves readability